modules/network-observability-netobserv-cli-install.adoc

```diff
@@ -18,24 +18,24 @@ You can optionally use Krew to install the `netobserv` CLI plugin. For more info
 
 .Procedure
 
-. Download the link:https://mirror.openshift.com/pub/openshift-v4/clients/netobserv/latest/[`oc netobserv` CLI tar file].
-. Unpack the archive:
+. Download the link:https://developers.redhat.com/content-gateway/rest/mirror2/pub/openshift-v4/clients/netobserv/latest[`oc netobserv` CLI tar file] that corresponds with your architecture.
+. Unpack the archive. For example, the `amd64` archive:
```

Comment:
```suggestion
. Unpack the archive. For example, for the `amd64` archive, run the following command:
```
Just a suggestion as it reads awkwardly to me, but it's late in the day so it could just be me. :)